File: common/org.daum.common.model/org.daum.common.model.root/src/main/java/org.daum.common.model.api/IModel.java
package org.daum.common.model.api;

import java.io.Serializable;

import org.daum.common.gps.api.IGpsPoint;

public interface IModel extends Serializable {

    public String getId();

    public void setId(String id);

	public void setLocation(IGpsPoint gpsPoint);

	public IGpsPoint getLocation();
}

File: common/org.daum.common.genmodel/org.daum.common.genmodel.root/src/main/scala/org.sitac/IModel.scala
package org.sitac;

trait IModel {
  private var location : org.sitac.GpsPoint = null

  def getId : java.lang.String

  def getLocation : org.sitac.GpsPoint ={
    this.location
  }

  def setLocation(location : org.sitac.GpsPoint) {
    this.location = location
  }
}